test.py

```diff
@@ -1,2 +1,7 @@
 def main():
-    pass
+    pass   
+
+
+print(A+    b)    
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test.py
+++ b/test.py
@@ -2,6 +2,6 @@
     pass   
 
 
-print(A+    b)    
+print(A+    b)
 
 f***!
```